tests/cypress/e2e/internal-push.test.js

```diff
@@ -57,6 +57,10 @@ describe( 'Internal Push', () => {
 			);
 
 			// Set Featured Image
+			// This test is temporarily disabled due to failures in GitHub actions that are not reproducible locally.
+			// On GitHub the success message expected within the cy.distributorPushPost command is not found as the
+			// distribution fails with an undefined error.
```

Comment:
Should we open an issue for this to tackle in the future?

Reply:
I've logged #1076 for follow up.